datalake-api/src/main/java/com/sequenceiq/sdx/api/endpoint/SdxEndpoint.java

```diff
@@ -183,6 +185,22 @@ SdxClusterResponse create(@ValidStackNameFormat @ValidStackNameLength @PathParam
     @ApiOperation(value = "list advertised datalake versions", produces = MediaType.APPLICATION_JSON, nickname = "advertisedruntimes")
     List<AdvertisedRuntime> advertisedRuntimes(@QueryParam("cloudPlatform") String cloudPlatform);
 
+    @POST
+    @Path("{name}/backupDatalake")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = "backup the datalake ", produces = MediaType.APPLICATION_JSON, nickname = "backupDatalake")
+    SdxBackupResponse backupDatalakeByName(@PathParam("name") String name,
+            @QueryParam("backupLocation") String backupLocation,
+            @QueryParam("backupName") String backupName);
+
+    @POST
+    @Path("{name}/backupDatalakeStatus")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = "backup status of the datalake ", produces = MediaType.APPLICATION_JSON, nickname = "backupDatalake")
```

Comment:
backupDatalakeStatus

Reply:
I see the nickname is backupDatalake for both the new api's added.

Reply:
addressed.